### PR TITLE
Make resetting API key when changing password optional

### PR DESCRIPTION
**Pull request by maartenba**
@harikmenon @skofman1 @xavierdecoster

### CODE REVIEW

File: src/NuGetGallery/ViewModels/AccountViewModel.cs [hunk 1]
```diff
@@ -11,6 +12,13 @@ namespace NuGetGallery
...
     public class AccountViewModel
     {
+        public AccountViewModel()
+        {
+            ChangePassword = new ChangePasswordViewModel
+            {
+                ResetApiKey = true
+            };
+        }
         public IEnumerable<string> CuratedFeeds { get; set; }
```
Comment (skofman1) [inline src/NuGetGallery/ViewModels/AccountViewModel.cs]: > p [](start = 7, length = 2)

line

**Comment by skofman1** [comment]
:shipit:

**Comment by xavierdecoster** [comment]
:shipit:

**Comment by harikmenon** [comment]
@maartenba are we all good to go for tomorrow then?

**Comment by harikmenon** [comment]
Thank You!!!!

**Pull request merged**